egs/wsj/s5/utils/lang/make_lexicon_fst_silprob.py

```diff
@@ -131,8 +131,9 @@ def read_lexiconp(filename):
     found_large_pronprobs = False
     # See the comment near the top of this file, RE why we use latin-1.
     with open(filename, 'r', encoding='latin-1') as f:
+        whitespace = re.compile("[ \t]+")
         for line in f:
-            a = line.split()
+            a = whitespace.split(line.strip())
```

Comment:
Thanks.  But we discovered a problem with doing line.strip(); it can remove character 255 I think which is part of the UTF-8 encoding range.  Can you change this and the other script to do line.rstrip('\n'), and test them?  (just to make sure they work in the normal case).

Reply:
Actually that's a more widespread problem, so I'll merge this and figure that problem out later.

Reply:
How about this way?
```python3
with open(filename, 'r', encoding='utf-8') as f:
        for line in f:
            a = [ p.encode().decode('latin-1') for p in line.split() ]
```
I think this way doesn't have the problem that you mentioned.